packages/fx-core/src/question/create.ts

```diff
@@ -1586,6 +1605,20 @@ export function openAIPluginManifestLocationQuestion(): TextInputQuestion {
   };
 }
 
+export function apiMessageExtensionAuthQuestion(): SingleSelectQuestion {
+  return {
+    type: "singleSelect",
+    name: QuestionNames.ApiMEAuth,
+    title: getLocalizedString("core.createProjectQuestion.apiMessageExtensionAuth.title"),
+    placeholder: getLocalizedString(
+      "core.createProjectQuestion.apiMessageExtensionAuth.placeholder"
+    ),
+    cliDescription: "The authentication type for the API.",
+    staticOptions: ApiMessageExtensionAuthOptions.all(),
+    default: "none",
```

Comment:
might be better to get value from option id directly rather than hard code here.

Reply:
Updated, thanks